Assets/VRTK/Scripts/Interactions/Highlighters/VRTK_MaterialColorSwapHighlighter.cs

```diff
@@ -29,8 +29,8 @@ public class VRTK_MaterialColorSwapHighlighter : VRTK_BaseHighlighter
 
         private Dictionary<string, Material[]> originalSharedRendererMaterials = new Dictionary<string, Material[]>();
         private Dictionary<string, Material[]> originalRendererMaterials = new Dictionary<string, Material[]>();
-        private Dictionary<string, Coroutine> faderRoutines;
-        private bool resetMainTexture = false;
+        protected Dictionary<string, Coroutine> faderRoutines;
```

Comment:
No, I made the MaterialPropertyBlock Colorswitcher an instance of the default MaterialColorSwapHighlighter, since the functionality is exactly the same, just the implementation of some methods differ (namely: `StoreOriginalMaterials` and `ChangeToHighlightColor`). Also, the options and inspector variables are exactly the same, thus some of the formerly private variables or functions had to become protected.